curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63fc85d4cc971ca3ecd547f2.md

```diff
@@ -0,0 +1,129 @@
+---
+id: 63fc85d4cc971ca3ecd547f2
+title: Step 1
+challengeType: 0
+dashedName: step-1
+---
+# --description--
+
+In this project, you will learn asynchronous JavaScript and how to use the `fetch` API. You will also learn how to display the data from an API on a web page. 
```

Comment:
While the `fetch` API is commonly used to interact with APIs, here we're just fetching a remotely stored JSON file. My understanding is that we're making a GET request to our CDN / network of servers, one of which responds with the JSON, but there's no real API to interact with in this project.

It might be best to be more general about `fetch` here and replace "endpoint" with "URL". While endpoints are all URLs, they're specific to APIs.

It might be better to use the term endpoint with the next project, which does involve interacting with the forum proxy API.

Reply:
THanks Kris. I searched through and replaced every occurrence of endpoint with URL